### DISCUSSION TITLE
terraform CI/CD pipeline, "Repository not found"

### DISCUSSION THREAD

**Discussion by jeffreymlewis** [gruntwork-io/knowledge-base#666] [Help] [answered]
Our Gruntwork CI/CD pipeline suddenly stopped working with errors like the following. Anyone else running into this? I'm thinking either github changed something, or my token(s) have expired.

```
[ecs-deploy-runner][2023-02-17T00:24:22+0000] 	* error downloading 'ssh://git@github.com/org/repo.git?ref=v0.94.1': /usr/bin/git exited with 128: Cloning into '/tmp/tmpltkuc0gw/dev/*********/dev/services/eks-cluster/repo/.terragrunt-cache/KT2kgJFUCMt8sVayfunMQ6a7wrg/MmsVrBnwzYe18iyiPxhzDEVSfKw'...
[ecs-deploy-runner][2023-02-17T00:24:22+0000] remote: Repository not found.
[ecs-deploy-runner][2023-02-17T00:24:22+0000] fatal: repository 'https://github.com/org/repo.git/' not found
```

For reference, this is trying to clone a private repo. I re-ran a workflow from yesterday, and the `plan` failed with the above error.

---

<ins datetime="2023-02-17T00:42:27Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109908">Tracked in ticket #109908</a></p>
</ins>

**Comment by jeffreymlewis** [comment]
Nope, github token is still valid. Hmm... 🤔 

```
curl -v -H "Authorization: token my-token-here" https://api.github.com/user/issues
```

**Reply by ellisonc** [reply]
Does the user still have access to the private repo?  If it is a gruntwork repo, access is controlled by our developer portal.

**Comment by jeffreymlewis** [comment, marked as answer]
All set now. One of my co-workers made a change, which broke something. (Not sure what he did, but it's working now!)
